MFVNeutralino/python/NtupleCommon.py

```diff
@@ -346,14 +355,37 @@ def ntuple_process(settings):
     else:
         return aod_ntuple_process(settings)
 
+def signal_uses_random_pars_modifier(sample): # Used for samples stored in inclusive miniaods
+    to_replace = []
+
+    if sample.is_signal:
+        if sample.name.startswith('ZH_') or sample.name.startswith('Wplus'):
+            magic_randpar = 'randpars_filter = False'
+            
+            decay = sample.name[sample.name.find('_')+1 : sample.name.find('_Z')]
+            # special case : 50um is referenced as 0p05 in config string
+            ctau = '%s' % (sample.tau/1000)
+            if ctau == '0.05' :
+                ctau = '0p05'
+            to_replace.append((magic_randpar, "randpars_filter = 'randpar %s M%i_ct%s-'" % (decay, sample.mass, ctau), 'tuple template does not contain the magic string "%s"' % magic_randpar))
+    return [], to_replace
+
 def signals_no_event_filter_modifier(sample):
     if sample.is_signal:
+        print("is signal")
         if use_btag_triggers :
-            magic = "event_filter = 'bjets OR displaced dijet'"
+            magic = "event_filter = 'bjets OR displaced dijet veto HT'"
```

Comment:
Did this intentionally get changed? I suspect the veto HT was in there a long time ago, but that Shaun changed it. Can we check if any accidental changes like this have made their way in anywhere else, too? Just a simple grep should find what we need. @pkotamnives @abbywarden @shaunhogan